clusterloader2/testing/density/suite.yaml

```diff
@@ -0,0 +1,8 @@
+- identifier: vanilla
+  configPath: testing/density/config.yaml
+  overridePaths: []
```

Comment:
Let's create a separate suite just for scheduler benchmarks. We'll run the vanilla as usual and start running the scheduler suite once a day or so.

Reply:
Done.